src/server/utils/getWebpackPublicPath.js

```diff
@@ -1,14 +1,18 @@
+import path from 'path';
+import protoRelUrl from './protoRelUrl';
```

Comment:
at the risk of longer function names, let's write this out, i have no idea what it means.